robosat/utils.py

```diff
@@ -20,3 +25,23 @@ def plot(out, history):
 
     plt.savefig(out, format="png")
     plt.close()
+
+
+def leaflet(out, base_url, tiles, grid, ext):
+
+    grid = json.dumps([mercantile.feature(tile) for tile in grid]) if grid else "''"
+
+    leaflet = open("./robosat/tools/templates/leaflet.html", "r").read()
```

Comment:
This works only when invoked from within the robosat directory, no?

Reply:
Interesting question !
As i didn't find anything related to setup/deploy so far, i still launch rs commands from the robosat directory.

Is there already an another way to do ?